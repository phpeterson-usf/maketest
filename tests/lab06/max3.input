$(repo_dir)/max3_main
