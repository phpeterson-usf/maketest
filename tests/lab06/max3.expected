max3_c(1, 2, 3) = 3
max3_s(1, 2, 3) = 3
max3_c(1, 99, 3) = 99
max3_s(1, 99, 3) = 99
max3_c(1, -2, -3) = 1
max3_s(1, -2, -3) = 1
